QUEST.tsv

```diff
@@ -2765,252 +2765,249 @@ QUEST_20151016_002772	$When the Royal Army said that they will spare our lives w
+QUEST_20151022_002874	There seems to be a fly hidden among us.. Find and eliminate them.
+QUEST_20151022_002875	Halt! You must not be an average fly to make it this far.
+QUEST_20151022_002876	Perfect timing, my body was starting to turn stiff.{nl}Don't you dare think you'll return alive. Pipsqueak!
+QUEST_20151022_002877	Don't you believe this is over.. If you think I'll let you rescue Lada, think again!
+QUEST_20151022_002878	I can't handle over the Kruvina to you!
+QUEST_20151022_002879	Since it's become to this, I will use the power of Kruvina to crush you!
+QUEST_20151022_002880	What are you doing.. Beholder..{nl}This is none of your concern!
+QUEST_20151022_002881	This is Giltine's power.{nl}Not something trifling person like you to desire. 
+QUEST_20151022_002882	In return, hand over the power that Giltine has approved.
+QUEST_20151022_002883	Thank you for rescuing us. If it weren't for your light, it might have become dangerous.{nl}I will return back to town and inform that the ferrets have appeared.
+QUEST_20151022_002884	Ferrets play dead in critical situations.
+QUEST_20151022_002885	Ferrets are curious animals and eats whatever they have yet to seen.
+QUEST_20151022_002886	My eyes met with the ferret who's been looking around. The ferret looks bored.
+QUEST_20151022_002887	Ferrets are nocturnal and seems to sleep a lot during the day.
+QUEST_20151022_002888	Ferrets are carnivorous animals and doesn't seem to like fruits that much.
+QUEST_20151022_002889	Demons. Set up. Totem.{nl}Totem. Touch. Ferrets become stronger.{nl}Ferrets. Win against humans!
+QUEST_20151022_002890	The fruit has become bigger. Run away coward humans.{nl}No human qualifications! Ferrets will scare them away!
+QUEST_20151022_002891	Demons. Helps the ferrets. Demons. Turned ferrets stronger.{nl}Demons. Tried to protect. Humans. Cannot come closer.{nl}Then demons. Continue to help the ferrets.
+QUEST_20151022_002892	This ferret doesn't hold a single interest to converse with me.
+QUEST_20151022_002893	The mysterious girl has run off to Zeraha without a word. Let's follow her. 
+QUEST_20151022_002894	Beef Jerky cannot be found here.
+QUEST_20151022_002895	Only fruits can be seen from this basket.
+QUEST_20151022_002896	This must be the wrong basket.
+QUEST_20151022_002897	The girl appeared and asked me to follow her.
+QUEST_20151022_002898	Black Soul
+QUEST_20151022_002899	I can feel the flow of enigmatic power after burning the black soul of the demons.
+QUEST_20151022_002900	Whispers can be heard to take them to the soul altar.
+QUEST_20151022_002901	I thought they would come to rescue their comrade, but..{nl}The wrong person seem to be caught.
+QUEST_20151022_002902	You seem to be doing well in my governed prison.{nl}You might be the Revelator that Giltine has spoke of.{nl}However, even the Revelator is merely a human.. I, Marnox cannot be resisted against!
+QUEST_20151022_002906	Acquired Bracken Herb.
```

Comment:
I believe you deleted some lines?
QUEST_20151022_002903
QUEST_20151022_002904
QUEST_20151022_002905 are missing